File: app/src/main/java/tourguide/tourguidedemo/NoOverlayActivity.java
package tourguide.tourguidedemo;

import android.app.Activity;
import android.os.Bundle;
import android.support.v7.app.ActionBarActivity;
import android.view.View;
import android.widget.Button;

import tourguide.tourguide.Pointer;
import tourguide.tourguide.ToolTip;
import tourguide.tourguide.TourGuide;


public class NoOverlayActivity extends ActionBarActivity {
    public TourGuide mTourGuideHandler;
    public Activity mActivity;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mActivity = this;
        setContentView(R.layout.activity_basic);

        final Button button1 = (Button) findViewById(R.id.button1);
        final Button button2 = (Button) findViewById(R.id.button2);
        // the return handler is used to manipulate the cleanup of all the tutorial elements
        mTourGuideHandler = TourGuide.init(this).with(TourGuide.Technique.Click)
                .setPointer(new Pointer()) // set pointer to null
                .setToolTip(new ToolTip().setTitle("Welcome :)").setDescription("Have a nice and fun day!"))
                .setOverlay(null)
                .playOn(button1);

        button1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                mTourGuideHandler.cleanUp();
            }
        });
        button2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                mTourGuideHandler.playOn(button1);
            }
        });
    }
}

File: app/src/main/java/tourguide/tourguidedemo/NoOverlayActivity.kt
package tourguide.tourguidedemo

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_basic.*
import tourguide.tourguide.Pointer
import tourguide.tourguide.ToolTip
import tourguide.tourguide.TourGuide


class NoOverlayActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_basic)

        val tourGuide = TourGuide.init(this).with(TourGuide.Technique.CLICK)
                .setPointer(Pointer()) // set pointer to null
                .setToolTip(ToolTip().setTitle("Welcome :)").setDescription("Have a nice and fun day!"))
                .playOn(button1)

        button1.setOnClickListener { tourGuide.cleanUp() }
        button2.setOnClickListener { tourGuide.playOn(button1) }
    }
}
